projects/storefrontapp/src/environments/b2b/b2b.feature.ts

```diff
@@ -18,7 +18,7 @@ export const b2bFeature: FeatureEnvironment = {
       featureModules: {
         organization: {
```

Comment:
Should we now call the cms feature `organization`? Maybe `organizationAdministration`?

Reply:
We agreed offline to do it in the next iteration.